pcair.R

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+
+argp <- arg_parser("Run PC-AiR") %>%
+  add_argument("gds_file", help = "GDS file") %>%
+  add_argument("kin_file", help = "Kinship matrix") %>%
+  add_argument("div_file", help = "") %>%
+  add_argument("--variant_id", help = "File with vector of variant IDs") %>%
+  add_argument("--sample_id", help = "File with vector of sample IDs") %>%
+  add_argument("--out_prefix", help = "Prefix for output files",
+               default = "") %>%
+  add_argument("--kin_thresh", help = "Kinship threshold for pcair",
+               default = 2 ^ (-9 / 2)) %>%
```

Comment:
@smgogarten Is this the correct value for 3rd degree? I wrote "3rd degree" in my notes but not the actual value for it smh

Reply:
yes, this is right